release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -397,3 +397,122 @@ Issued: 2021-xx-xx
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Log forwarding
+** Precision Time Protocol (PTP) hardware
+** CSI volume snapshots
+
+* The following {product-title} features are unsupported:
+** {product-title} Virtualization
+** Red Hat OpenShift Service Mesh
+** CodeReady Containers (CRC)
+** {product-title} Metering
+** Multus CNI plug-in
+** FIPS cryptography
+** Encrypting data stored in etcd
+** Automatic repair of damaged machines with machine health checking
+** Tang mode disk encryption during {product-title} deployment
+** {product-title} Serverless
+** Helm command-line interface (CLI) tool
+** Controlling overcommit and managing container density on nodes
+** etcd cluster Operator
+** CSI volume cloning
+** NVMe
+
+* Worker nodes must run {op-system-first}.
+* Persistent shared storage must be provisioned by using either NFS or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or LSO with DASD/FCP.
+* These features are available only for {product-title} on IBM Z for {product-version}:
+** HyperPAV enabled on IBM System Z /LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+===== Supported features
+
+With this release, the following features are supported on IBM Z and LinuxONE:
+
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* OpenShift Do (odo)
+* OpenShift Pipelines TP
+* Persistent storage using Fibre Channel
+* Raw Block
+* z/VM Emulated FBA devices on SCSI disks
+* Multi-path Support
+* OVNKube
+
+[id="ocp-4-7-ibm-power"]
+==== IBM Power Systems
+
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power Systems] or xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power Systems in a restricted network].
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power Systems:
+
+* {product-title} for IBM Power Systems does not include the following Technology Preview features:
++
+** OpenShift Virtualization
+** OpenShift Serverless (knative, FaaS integrations)
+
+* The following {product-title} features are unsupported:
+** Red Hat OpenShift Service Mesh (istio, jaeger, kiali)
```

Comment:
Will need to check the new date for Service Mesh. If this is before the 4.7 GA, we should mark Service Mesh as supported.

Reply:
@manojnkumar The plan is that Service Mesh will go GA with OCP 4.6. Therefore, I would leave it as unsupported right now. 

cc @tgargya

Reply:
@SNiemann15 : Is the service mesh GA captured elsewhere in the documentation?

Reply:
@manojnkumar Yes we have it in Tanias PR for Service Mesh: https://github.com/openshift/openshift-docs/pull/28313